doc/build-helpers/images/dockertools.section.md

```diff
@@ -1486,6 +1494,100 @@ The environment in the image doesn't match `nix-shell` or `nix-build` exactly, a
 
   _Default value:_ `null`.
 
+`includeBuildDerivation` (Boolean; _optional_)
+
+: Whether to include a `buildDerivation` binary which builds the derivation.
+
+  _Default value:_ `true`.
+
+`extraContents` (Path or List of Paths or Null; _optional_)
+
+: Extra directories whose contents will be added to the generated image.
+  Things that coerce to paths (e.g. a derivation) can also be used.
+  This can be seen as an equivalent of `ADD extraContents/ /` in a `Dockerfile`.
+
+  If set to `null`, some convenience contents are included:
+
+  - `/bin/sh`
+    - Symlink to `${bashInteractive}/bin/bash`.
+  - `/usr/bin/env`
+    - Symlink to `${coreutils}/bin/env`.
+
+  _Default value:_ `null`.
```

Comment:
How is the list of paths handled?
Is it like multiple `ADD`s, or does it produce a link farm and add that? Or is it always a link farm?